### PR TITLE
Circleci update resource class

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,149 +9,273 @@ jobs:
...
+          command: BASE_IMAGE_TAG=focal-3.8 make build
+      - run:
+          name: Tag and save docker image
+          command: |
+            docker tag geotrek:latest geotrekce/circleci-admin:$CIRCLE_WORKFLOW_ID
+            docker save -o ./20.04.$CIRCLE_WORKFLOW_ID.tar geotrekce/circleci-admin:$CIRCLE_WORKFLOW_ID
+      - persist_to_workspace:
+          root: .
+          paths:
+            - 20.04.*.tar
```
Comment (LePetitTim) [inline .circleci/config.yml:60]: Pourquoi tu reutilises pas $CIRCLE_WORKFLOW_ID ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,149 +9,273 @@ jobs:
...
+    machine:
+      image: ubuntu-2004:202201-02
     environment:
       LANG: C.UTF-8
       ENV: tests
       SECRET_KEY: foo
       CUSTOM_SETTINGS_FILE: ""
       CONVERSION_HOST: localhost
       CAPTURE_HOST: localhost
-    working_directory: /opt/geotrek-admin
```
Comment (LePetitTim) [inline .circleci/config.yml:119]: Why did it exist before ?

File: .circleci/config.yml (hunk 1 above)
Comment (submarcos) [reply .circleci/config.yml:60]: no matter WORKFLOW ID because persist_to_workspace is unique by workflow

Comment (submarcos) [reply .circleci/config.yml:60]: and it's not possible to use ENV variable here

File: .circleci/config.yml (hunk 2 above)
Comment (submarcos) [reply .circleci/config.yml:119]: before, previous built docker image was used directly by CIrcle. Now, we load image and launch it. 
In bot cases, /opt/geotrek-admin is always default working_directory

File: .circleci/config.yml (hunk 1 above)
Comment (submarcos) [reply .circleci/config.yml]: fixed by using $CIRCLE_SHA1 (allow to relaunch failed job) and not specify ID in saved image

**Pull request closed without merging**